### ISSUE TITLE
Yaourt accepts wrong input when user is asked to continue upgrade

### ISSUE THREAD

**Issue by vwugd97** [archlinuxfr/yaourt#218]
Hei, 

I just noticed that yaourt (Version: 1.7) asks for "Y" or "n" to continue package upgrades but continues even if the user inputs other characters like "A" or "<" however this does not happen when the user is asked if he wants to continue the package installation.

Screenshot:
![alt text](http://i.imgur.com/sZTbLHb.png "Wrong input error")

The problem seems to be in the "abs.sh.in" file (https://github.com/archlinuxfr/yaourt/blob/master/src/lib/abs.sh.in). Since both functions "display_update" and "show_targets" ask for user input and the problem does not happen in "show_targets" changing 

```display_update() {
	...
		echo
		msg "$(gettext 'Continue upgrade ?') $(yes_no 1)"
	...
}```

to

```display_update() {
	...
		echo
		prompt "$(gettext 'Proceed with upgrade? ') $(yes_no 1)"
	...
}```

may fix the problem?

**Comment by larchunix** [comment]
IIUC, in your screenshot the first prompt is from `yaourt` (`display_update()`) and the second one is from `pacman`. `show_target()` function is not called.

`yaourt`'s behaviour is not consistent with `pacman`, but I don't know if it was coded like this intentionally or not.

`yaourt` considers that the default answer is `Y` and unknown answers are treated like `Y`.

**Comment by f2404** [comment]
Should we align the behavior to `pacman`?

**Comment by larchunix** [comment]
I guess so.

**Comment by larchunix** [comment]
I just noticed this issue is the same as #37.

**Comment by larchunix** [comment]
I have opened a PR which should fix this issue. Could you test #230 please ?

**Comment by Skunnyk** [comment]
Yes, we should align to pacman behavior :+1:

**Comment by larchunix** [comment]
Fixed by #230.
